http/src/v0/version.rs

```diff
@@ -17,6 +17,12 @@ pub struct Response {
     version: &'static str,
     commit: &'static str,
     // repo is here for go-ipfs and js-ipfs but we do not have full repo at the moment
+    // empty string is accepted by the tests
+    repo: &'static str,
+    // required by rust-ipfs-api as of 0.7.0 at least
+    system: &'static str,
+    // required by rust-ipfs-api as of 0.7.0 at least
+    golang: &'static str,
```

Comment:
That means rust-ipfs-api should be fixed, I don't think it's a good idea to add a golang version? Does the js-ipfs do that?

Reply:
Future versions could be fixed but I was leaning towards "people might already have code ready" using any number of released versions of `rust-ipfs-api`. Granted we are far away from being a drop-in replacement but I thought people might be interested to test.

> Does the js-ipfs do that?

Nope, just the tested 3 properties. https://github.com/ipfs/js-ipfs/blob/master/packages/ipfs/src/http/api/resources/version.js

Could be that it's better not to start with a lot of workarounds.

Reply:
Created issue on rust-ipfs-api, resolving.